src/test/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceTest.java

```diff
@@ -117,9 +125,50 @@ public void testRestrictionFromQuestions() throws IOException {
         assertTrue(res.equals("{\"useRestriction\":\"Manual Review\"}"));
     }
 
+    @Test
+    public void testDarFound() throws Exception {
+        // Create a DAR
+        Client client = ClientBuilder.newClient();
+        Document sampleDar = DataRequestSamplesHolder.getSampleDar();
+        checkStatus(CREATED, post(client, darPath(), sampleDar));
+        List<Document> created = retrieveDars(client, darPath());
+        System.out.println("Created Size: " + created.size());
+        assertEquals(1, created.size());
+        Document newDoc = created.get(0);
+
+        // TODO: Figure out how to
```

Comment:
@dags - Here (and below) is where I'm struggling. I can't figure out how to get the hex string value of the saved Data Access Request id. I know it's there, it looks something like 
```
{timestamp=1553890382, machineIdentifier=6733188, processIdentifier=17821, counter=13091050, timeSecond=1553890382, time=1553890382000, date=1553890382000}
```
But ... how do I turn that into the ObjectId.toString value which is what we would use in the API call to get a DAR by id? I can't find anything in the docs or in stack overflow that address this. 

Thank you!

Reply:
Ok, I think I've figured out something super painful. This gives me a real hex string from manually constructing the ObjectId from the json version of the document's `_id`. This is super ugly, painful, and should not be necessary.
```
        Document newDoc = created.get(0);
        LinkedHashMap id = (LinkedHashMap) newDoc.get(DarConstants.ID);
        ObjectId objectId = new ObjectId(
                Integer.valueOf(id.get("timestamp").toString()),
                Integer.valueOf(id.get("machineIdentifier").toString()),
                Short.valueOf(id.get("processIdentifier").toString()),
                Integer.valueOf(id.get("counter").toString())
        );
```